integration/src/test/resources/org/mule/test/el/expression-language-config.xml

```diff
@@ -15,7 +15,7 @@
     <configuration>
         <expression-language autoResolveVariables="false">
             <import name="loc" class="java.util.Locale" />
-            <import class="org.mule.runtime.core.util.StringUtils" />
+            <import class="org.apache.commons.lang.StringUtils" />
```

Comment:
don't you need commons-lang as a compile dependency in the test module for this to work?